spec/lyber_core/robots/robot_spec.rb

```diff
@@ -1,103 +1,137 @@
-require_relative 'test_robot'
-require_relative 'test_robot_with_skip'
-require_relative 'test_robot_with_note'
-require_relative 'test_robot_with_note_and_skip'
-require_relative 'test_robot_with_constant_state'
+describe 'robot "bases"' do
+  let(:druid) { 'druid:test1234' }
+  let(:wf_name) { 'testWF' }
+  let(:step_name) { 'test-step' }
+  let(:test_robot_with_skip) do
+    Class.new(test_robot) do
+      def perform(_druid)
+        super && LyberCore::Robot::ReturnState.new(status: 'skipped')
+      end
+    end
+  end
+  let(:test_robot_with_note) do
```

Comment:
if these are only used in one place, i'd be inclined to put them inline with that test case so I don't have to scroll around.

Reply:
That's not what rubocop-rspec seems to prefer.